jsdrupal/drupal-admin-ui
# [#599] Replace Apache with nginx and run services as non-root, with t…

…he new users having the same UID/GID as the host user to ensure correct volume permissions

## Issue

https://github.com/jsdrupal/drupal-admin-ui/issues/599

## Testing instructions

- Remove any untracked files (`git clean -fdX`)
- Rebuild the docker containers with the new command mentioned in the README
- Bring up the containers, enter them and check mounted volumes and their files are using the non-root user
- Outside of the containers, check the files created in mounted volumes are owned by your host user
